.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Kubernetes-sigs/krew CI
+on: 
+  push:
+    branches: 
+      - '*'
+    tags: 
+      - 'v*.*.*'
+  pull_request:
+    branches: 
+      - '*'
+  release:
+    types: [published, created, edited]
+
+jobs:
+
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+    
+    - name: Set GOPATH
+      # temporary fix
+      # see https://github.com/actions/setup-go/issues/14
+      run: |
+        set -euo pipefail
+        echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+        echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+    
+    - name: Install dependencies
+      run: go mod download
+    
+    - name: Install gox
+      run: hack/install-gox.sh
+      
+    - name: Verify code quality
```

Comment:
What if we just separated each of these commands to separate `steps`?

Because in this mode, I can't tell (from the output) which one is failing.

In travis we had that.

Since the script names are quite self descriptive, you don't even need to specify `name:` for most.

Reply:
> Because in this mode, I can't tell (from the output) which one is failing.
> 
I don't like the idea of separating all steps for the dead times when it changes of step. But, I agree with this. I just separated all the steps, check it, please.